midi_app_controller/controller/connected_controller.py

```diff
@@ -28,8 +29,16 @@ class ConnectedController:
         A list containing all valid button ids on a handled controller.
     knob_ids : List[int]
         A list containing all valid knob ids on a handled controller.
-    knob_engagement: Dict[int, int]
+    knob_engagement : Dict[int, int]
         A dictionary that keeps the value of every knob.
+    butons_mutex : QMutex
+        Mutex for worker threads.
+    flashing_buttons : set[int]
+        Set of the button ids, that are currently flashing.
```

Comment:
slightly ungrammatical, maybe "Set with ids of buttons that are currently flashing"?